xrally_kubernetes/env/platforms/existing.py

```diff
@@ -129,3 +129,98 @@ def _get_validation_context(self):
     def info(self):
         version = k8s_service.Kubernetes(self.platform_data).get_version()
         return {"info": version}
+
+    @classmethod
+    def create_spec_from_sys_environ(cls, sys_environ):
+        """Create a spec based on system environment.
+
+        The environment variables could be defined with two mutually exclusive
+        mandatory ways: defining certificates or defining auth token.
+
+        To define certificates to connect next variables used:
+        .. envvar:: KUBERNETES_HOST
```

Comment:
1) list items should be splitted from L140 by an empty line.
2) it looks like a wrong indention. List items should have +2 spaces to the number of spaces of the list definition(i.e L140 line)